README.rst

```diff
@@ -156,10 +150,26 @@ line, run:
 This will download the AAPL price data from `quantopian-quandl` in the
 specified time range and stream it through the algorithm and save the
 resulting performance dataframe to dma.pickle which you can then load
-and analyze from within python.
+and analyze from within Python.
 
 You can find other examples in the ``zipline/examples`` directory.
 
+Questions?
+==========
+
+If you find a bug feel free to `open an issue <https://github.com/quantopian/zipline/issues/new>`_ and fill out the issue template.
+
+Contributing
+============
+
+All contributions, bug reports, bug fixes, documentation improvements, enhancements and ideas are welcome. Details on how to set up a development environment can be found in our `development guidelines <http://zipline.io/development-guidelines.html>`_
```

Comment:
Missing a period at the end. Also, I'm a stickler for the oxford comma after "enhancements", if you please.